src/backend/looped/vehicle/navigation.cpp

```diff
@@ -0,0 +1,55 @@
+#include "backend/looped_command.hpp"
+#include "fiber_pool.hpp"
+#include "gta/enums.hpp"
+#include "natives.hpp"
+#include "pointers.hpp"
+#include "util/blip.hpp"
+#include "util/entity.hpp"
+#include "util/vehicle.hpp"
+
+
+namespace big
+{
+	class navigationvoice : looped_command
+	{
+		//bool needsNextDirection = true;
+		using looped_command::looped_command;
+
+		virtual void on_enable() override
+		{
+			g_notification_service->push_success("Voice Navigation", "Voice Navigation has been enabled");
+		}
+
+		virtual void on_tick() override
+		{
+			if (g.self.nav_voice)
+			{
+				if (g_player_service->get_self()->get_current_vehicle() != nullptr)
+					g_fiber_pool->queue_job([] {
```

Comment:
This is stupid, why are you queueing a job from within a tick, which runs inside a script thread?

Reply:
Because I call a native and audio functions.

Reply:
https://github.com/YimMenu/YimMenu/blob/44f79a428495bf4b63a6e905a5574e6cc3011a36/src/backend/backend.cpp#L42

looped commands already run inside a script, you don't need `queue_job`

Reply:
Ok, I will change this. In main.cpp do i need to add it to the script manager?